pages/apis/rest_api/analytics/flaky_tests.md.erb

```diff
@@ -0,0 +1,32 @@
+# Flaky tests API
+
+{:toc}
+
+This feature is currently in beta, and not available to all organizations. If you would like early access, please TBC.
+
+## List flaky tests
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of the user's flaky tests.
+
+```bash
+curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{suite.slug}/flaky-tests?access_token={token}"
```

Comment:
Is including the access token in a query parameter standard for Buildkite? I see in the overview that it only mentions using the `Authorization` header.

Reply:
It can be added as a query param;  I'll remove it here though as it's not a pattern included in our other documentation